docker/Dockerfile.pdo-tp-ccf

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+# Description:
+#   Build Container for PDO CCF Transaction Processor
+#
+# Build:
+#   $ docker build --build-arg host_ip=<host-ip-address> -f Dockerfile.pdo-tp-ccf -t pdo-tp-ccf .
+#
+# Run:
+#   $ docker run --net=host --expose 6600 -it pdo-tp-ccf
+#
+# A CCF instance will be availble for business transctions at http://host-ip:6600.
+#
+# If you are behind proxy, (and if building the image using this dockerfile rather than docker-compose),
+# make sure that proxy settings are set in the host machine @ ~/.docker/config.json
+# The settings from this file will be inherited by the container. See https://docs.docker.com/network/proxy/ for details.
+
+# we start from ccf base container that comes preinstalled with ccf base and the depdencies for building the application
+FROM ccfciteam/ccf-app-ci:0.11.7
+
+#install a few extra utilities
+RUN apt-get update && apt-get install -y python3-pip vim net-tools
+RUN pip3 install virtualenv
+
+ARG PDO_REPO_URL=https://github.com/hyperledger-labs/private-data-objects.git
+ARG PDO_REPO_BRANCH=master
```

Comment:
From a readability perspective, would be nicer to have build arguments right away after the FROM and document the parameters in the header; but then i sinned already regarding the former aspect with `Dockerfile.pdo-tp` ...